netbox/project-static/styles/custom/_misc.scss

```diff
@@ -33,3 +33,11 @@ span.color-label {
 .netbox-edition {
   letter-spacing: .15rem;
 }
+
+// A floating div for form buttons
+.btn-float-group {
+  position: sticky;
+  bottom: 0px;
+  z-index: 1;
+  padding: 2px 0px 10px 0px;
```

Comment:
We may want to add a bit more top padding. IMO vertical spacing should be the same above and below the buttons while floating.

Reply:
I went a different direction here. Let me know what you think. Essentially I removed the button group `div` background altogether, which created truly 'floating' buttons and IMO added more real estate to the form by not covering it up.
 
![NetBox Button Illustration Rev](https://github.com/user-attachments/assets/4be41340-2da6-46b5-8bad-111bdb614cb8)